README.md

```diff
@@ -54,6 +54,12 @@ The Python SIG meets weekly on [Thursday 9 AM PDT](https://calendar.google.com/c
 
 You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-python).
 
+### Auto-Instrumentation (cross-language)
+
+"Auto-Instrumentation" refers to efforts to install OpenTelemetry instrumentation and otherwise extract OpenTelemetry-compatible data from processes without direct code modification. The Auto-Instrumentation SIG will meet weekly at a time TBD.
+
+You can also join us on [the auto-instrumentation channel](https://gitter.im/open-telemetry/auto-instrumentation) in OpenTelemetry gitter.
```

Comment:
The channel seems to be private, doesn't load for me. Is it invite-only?

Reply:
I mentioned this above. I haven't created the channel yet... I will do so if this PR is approved.